server/accesslog.go

```diff
@@ -194,6 +201,8 @@ func (a *AccessLogServer) handleHTTPLogs(msg *als.StreamAccessLogsMessage_HttpLo
 			ResponseStatusCode:           responseCode,
 			GatewaySource:                a.gatewaySource,
 			ClientIP:                     req.GetForwardedFor(),
+			GrpcStatusCode:               v.GetResponse().GetResponseHeaders()["grpc-status"],
```

Comment:
These failures seem to indicate you need to update the go.mod? Please try it again.

Reply:
Nevermind, I see that this must have been a stale error message. Please ignore.